prog/dftb+/include/common.fypp

```diff
@@ -78,4 +78,12 @@
 #{if rank == 0}##{else}#(${":" + ",:" * (rank - 1)}$)#{endif}#
 #:enddef
 
+
+#! Macro for automated deallocations
+#:def DEALLOCATE(*args)
```

Comment:
The macro name is a little misleading, how about SAFEALLOC or REALLOCATE?

Reply:
The macro is a simple conditional deallocation and does not contain any allocation. But I have renamed it in 671f0dd to `SAFE_DEALLOC` to express the intention more precisely.